packages/terra-site/src/examples/content-container/ContentContainerFill.jsx

```diff
@@ -22,11 +22,12 @@ Phasellus in eleifend odio. Nunc maximus sem vel odio lobortis, in accumsan arcu
 pharetra, augue felis aliquet nunc, non facilisis est velit vel elit.
                  </p>);
 
-const contentheader = <Heading level={1} size={'large'}>Display Header</Heading>;
+const contentheader = <Heading level={3}>Display Header</Heading>;
+const contentfooter = <Heading level={3}>Display Footer</Heading>;
 
 const container = () => (
   <div style={{ height: '200px', overflow: 'auto' }}>
```

Comment:
Can you throw a border around these examples? Its difficult to tell these site examples from the the example label. :)

Reply:
Thanks, that was bothering me too 8219786d349bee8ea577c702a65c5994eaf6cf32